section1/reflection.md

```diff
@@ -1,19 +1,31 @@
 ## Section 1 Reflection
```

Comment:
Really nice work all around. In your mentor DTR you mentioned being detail oriented and it shows - your code is consistently clean and follows Ruby conventions.